conda-store-server/conda_store_server/dbutil.py

```diff
@@ -71,7 +73,11 @@ def upgrade(db_url, revision="head"):
     revision: str [default: head]
         The alembic revision to upgrade to.
     """
-    engine = create_engine(db_url)
+    engine = create_engine(
+        db_url,
+        # See the comment on the CustomJSONEncoder class on why this is needed
+        json_serializer=lambda x: json.dumps(x, cls=utils.CustomJSONEncoder),
```

Comment:
I think this could also be a `functools.partial` like

```suggestion
        json_serializer=partial(json.dumps, cls=utils.CustomJSONEncoder),
```

... in case the lambda feels awkward.